examples/LaserWakefield/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -120,41 +128,51 @@ typedef Particles<ParticleDescription<
 struct Hydrogen
 {
     static const float_X numberOfProtons  = 1.0;
-    static const float_X numberOfNeutrons = 0.0;
+    static const float_X numberOfNeutrons = 1.0;
```

Comment:
that's intended. deuterium has the same charge/mass ratio as carbon, helium, oxygen, ... in their natural isotopes. protons not.